### ISSUE TITLE
Guidance/component for hint text needed

### ISSUE THREAD

**Issue by sarahpulis** [govau/design-system-components#598]
# Feature Request
- [ ] I’ve read and understood the [Contributing guidelines](https://github.com/govau/design-system-components/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [ ] I’ve read and agree to the [Code of Conduct](https://github.com/govau/design-system-components/blob/master/CODE_OF_CONDUCT.md).
- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.
- [ ] I've searched the [community forums](https://community.digital.gov.au/c/designsystem) to check for related issues or feature requests

I'd like to request that guidance on hint text for from controls is added to the design system. This is a commonly needed feature and it would be great to have consistency and predictability.

Some things to consider: 
* visual location of hint text for form controls and for group of form controls
* when/if short hint text should be part of the form label
* associating hint text with form control for screen readers

**Comment by alex-page** [comment]
Thanks @sarahpulis the forms are missing a lot of components. This is a great one and a win for a11y.

**Comment by sarahpulis** [comment]
No probs @alex-page, just putting in my wishlist based on how I've seem components used on some of the DTA sites. :-)

**Comment by alex-page** [comment]
That is great, we will need this component for form templates as well!

**Comment by alex-page** [comment]
@sukhrajghuman before commencing work look at how some popular frameworks do this. I think we should slightly refactor the text input so that it is more robust.

**Comment by sukhrajghuman** [comment]
Hey @sarahpulis thanks for bringing this up! I'm going to merge this ticket with #623 😄
